clerk/clerk.go

```diff
@@ -26,6 +26,9 @@ type Client interface {
 	NewRequest(method string, url string, body ...interface{}) (*http.Request, error)
 	Do(req *http.Request, v interface{}) (*http.Response, error)
 
+	DecodeToken(token string) (*Claims, error)
+	VerifyToken(token string) (*Claims, error)
```

Comment:
Should we perhaps document those? We should explicitly state that decode does not verify the token.